diff --git a/node_modules/prompts/lib/elements/autocompleteMultiselect.js b/node_modules/prompts/lib/elements/autocompleteMultiselect.js
index 16ec459..5901c3c 100644
--- a/node_modules/prompts/lib/elements/autocompleteMultiselect.js
+++ b/node_modules/prompts/lib/elements/autocompleteMultiselect.js
@@ -123,13 +123,19 @@ class AutocompleteMultiselectPrompt extends MultiselectPrompt {
   }
 
   renderInstructions() {
-    return `
+    if (this.instructions === undefined || this.instructions) {
+      if (typeof this.instructions === 'string') {
+        return this.instructions;
+      }
+      return `
 Instructions:
     ${figures.arrowUp}/${figures.arrowDown}: Highlight option
     ${figures.arrowLeft}/${figures.arrowRight}/[space]: Toggle selection
     [a,b,c]/delete: Filter choices
     enter/return: Complete answer
-    `
+`;
+    }
+    return '';
   }
 
   renderCurrentInput() {
